src/core/ipv4/ip4_napt.c

```diff
@@ -222,6 +224,7 @@ ip_napt_enable(u32_t addr, int enable)
   } else {
     ip_napt_deinit();
   }
+  LWIP_DEBUGF(NAPT_DEBUG, ("ip_napt set to %d\n", enable));
```

Comment:
just to be in line with other lwip code
```suggestion
  LWIP_DEBUGF(NAPT_DEBUG, ("ip_napt set to "U16_F"\n", enable));
```